### DISCUSSION TITLE
Placeholders for ISLAND MISSIONS completed

### DISCUSSION THREAD

**Discussion by AquariamcDev** [BG-Software-LLC/SuperiorSkyblock2#2186] [Pending Feature Requests] [answered]
Can you add a ISLAND version of this please? %superior_player_missions_completed_<category>% (%superior_island_missions_completed_<category>%)

https://github.com/BG-Software-LLC/SuperiorSkyblock2/discussions/1104?sort=old

**Comment by OmerBenGera** [comment, marked as answer]
Added in latest dev build :)
